Source/SelfService/Web/stores/microservice.ts

```diff
@@ -153,9 +153,9 @@ const saveMicroservice = async (kind: string, input: any): Promise<boolean> => {
             return Promise.resolve(false as boolean);
     }
 
-    if (!response) {
-        alert('Sad times');
-        return Promise.resolve(false);
+    const jsonResponse = await response.json();
+    if (!response.ok) {
+        return Promise.reject(jsonResponse.message);
```

Comment:
I see that this was already done before you some places in this function, but since this function is `async` there is no need to wrap the return values in Promises, the compiler does that for you. For this code, the return type is actually a `Promise<Promise<any>>`.

Reply:
Removed the extra `Promise` stuff